packages/backend-output-schemas/src/storage/v1.ts

```diff
@@ -1,12 +1,6 @@
 import { z } from 'zod';
 
-const storageAccessActionEnum = z.enum([
-  'read',
-  'get',
-  'list',
-  'write',
-  'delete',
-]);
+const storageAccessActionEnum = z.enum(['get', 'list', 'write', 'delete']);
```

Comment:
wouldn't this be a breaking change for deployed apps ?

Reply:
You're right, I forgot to think about impact on existing apps, adding this back in.